strongbox-security/strongbox-user-management/src/test/java/org/carlspring/strongbox/users/service/UserServiceTest.java

```diff
@@ -243,10 +248,52 @@ public void testThatUserNameIsUnique()
     @Test
     public void testPrivilegesProcessingForAccessModel()
     {
+        String testUserName = "test-user";
+
+        UserDto userAdd = new UserDto();
+        userAdd.setEnabled(true);
+        userAdd.setUsername(testUserName);
+        userAdd.setPassword("test-password");
+        userAdd.setSecurityTokenKey("before");
+
+        UserAccessModelDto userAccessModelDto = new UserAccessModelDto();
+        UserStorageDto userStorageDto = new UserStorageDto();
+        userStorageDto.setStorageId("storage0");
+        UserRepositoryDto userRepositoryDto = new UserRepositoryDto();
+        userRepositoryDto.setRepositoryId("releases");
+        PrivilegeDto repositoryPrivilege = new PrivilegeDto("ARTIFACTS_RESOLVE","ARTIFACTS_RESOLVE");
+        userRepositoryDto.getRepositoryPrivileges().add(repositoryPrivilege);
+        UserPathPrivilegesDto userPathPrivilegesDto = new UserPathPrivilegesDto();
+        userPathPrivilegesDto.setPath("com/carlspring");
+        userPathPrivilegesDto.setWildcard(true);
+        PrivilegeDto pathPrivilege1 = new PrivilegeDto("ARTIFACTS_VIEW","ARTIFACTS_VIEW");
+        userPathPrivilegesDto.getPrivileges().add(pathPrivilege1);
+        userRepositoryDto.getPathPrivileges().add(userPathPrivilegesDto);
+        UserPathPrivilegesDto userPathPrivilegesDto2 = new UserPathPrivilegesDto();
+        userPathPrivilegesDto2.setPath("org/carlspring");
+        userPathPrivilegesDto2.setWildcard(true);
+        PrivilegeDto pathPrivilege2 = new PrivilegeDto("ARTIFACTS_DELETE","ARTIFACTS_DELETE");
+        userPathPrivilegesDto2.getPrivileges().add(pathPrivilege2);
+        userRepositoryDto.getPathPrivileges().add(userPathPrivilegesDto2);
+        UserPathPrivilegesDto userPathPrivilegesDto3 = new UserPathPrivilegesDto();
+        userPathPrivilegesDto3.setPath("com/mycorp");
+        PrivilegeDto pathPrivilege3 = new PrivilegeDto("ARTIFACTS_DELETE","ARTIFACTS_DELETE");
```

Comment:
Do you need all these objects, or could you just have these as nested variables?

Reply:
hmm what do you mean by nested variables?

Reply:
I mean instead of:
```
PrivilegeDto pathPrivilege3 = new PrivilegeDto("ARTIFACTS_DELETE","ARTIFACTS_DELETE");
PrivilegeDto pathPrivilege4 = new PrivilegeDto("ARTIFACTS_VIEW","ARTIFACTS_VIEW");
PrivilegeDto pathPrivilege5 = new PrivilegeDto("ARTIFACTS_DEPLOY","ARTIFACTS_DEPLOY");
PrivilegeDto pathPrivilege6 = new PrivilegeDto("ARTIFACTS_COPY","ARTIFACTS_COPY");

userPathPrivilegesDto3.getPrivileges().add(pathPrivilege3);
userPathPrivilegesDto3.getPrivileges().add(pathPrivilege4);
userPathPrivilegesDto3.getPrivileges().add(pathPrivilege5);
userPathPrivilegesDto3.getPrivileges().add(pathPrivilege6);
```
To use:
```
userPathPrivilegesDto3.getPrivileges().add(new PrivilegeDto("ARTIFACTS_DELETE","ARTIFACTS_DELETE"));
userPathPrivilegesDto3.getPrivileges().add(new PrivilegeDto("ARTIFACTS_VIEW","ARTIFACTS_VIEW"));
userPathPrivilegesDto3.getPrivileges().add(new PrivilegeDto("ARTIFACTS_DEPLOY","ARTIFACTS_DEPLOY"));
userPathPrivilegesDto3.getPrivileges().add(new PrivilegeDto("ARTIFACTS_COPY","ARTIFACTS_COPY"););
```

Unless you'll be re-using `pathPrivilege3` ... `pathPrivilege3` elsewhere as well.

This makes the boilerplate code a lot less to have to read through.

Reply:
ok, got it. makes sense. I will update the code accordingly. Thanks! :)

Reply:
Great! Thank you!

I know this is trivial and cosmetic, but it makes the code look much better on the eye and this saves time when reading through pull requests and codebases.

Reply:
done!